.circleci/config.yml

```diff
@@ -70,16 +101,8 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - publish_github:
-          context: Honeycomb Secrets
-          filters:
-            tags:
-              only: /^v.*/
-            branches:
-              ignore: /.*/
-          requires:
-            - test
-      - publish_github:
+      - build_extension
+      - publish_s3:
```

Comment:
It looks like you've replaced the `publish_github` command in place of `publish_s3` here, we want to do both right?

Reply:
No, publish_github isn't working at the moment because of a token permissions problem, so I've removed it for now. We only want publish_s3 for now.